### PR TITLE
Chapter1/homework/calculator

### PR DESCRIPTION
**Pull request by San-cat**
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter num1: ");
        int num1 = scanner.nextInt();

        System.out.print("Enter num2: ");
        int num2 = scanner.nextInt();

        System.out.print("Enter an operator (+, -, *, /): ");
        char operator = scanner.next().charAt(0);

       int result = 0;

        switch(operator)
        {
            case '+':
                result = num1 + num2;
                break;

            case '-':
                result = num1 - num2;
                break;

            case '*':
                result = num1 * num2;
                break;

            case '/':
                if (num2 !=0) {
                    result = num1 / num2;
                } else {
                    result = -1;
                }
                break;

        }

        System.out.println(result);

    }
}

### CODE REVIEW

**Comment by domiiii11** [comment]
I think the code doesn't meet all requirements, because according to the task you have to take input as a string whole line "passing a string "5+5". In this case, the program accepts three inputs two numbers as integers and one symbol as a char.

**Comment by Almantask** [comment]
> I think the code doesn't meet all requirements, because according to the task you have to take input as a string whole line "passing a string "5+5". In this case, the program accepts three inputs two numbers as integers and one symbol as a char.

that's correct- the code should meet the minimum requirements- which is the method definition

**Comment by Almantask** [comment]
> import java.util.Scanner;
> 
> public class Main {
> public static void main(String[] args) {
> 
> ```
>     Scanner scanner = new Scanner(System.in);
>     System.out.print("Enter num1: ");
>     int num1 = scanner.nextInt();
> 
>     System.out.print("Enter num2: ");
>     int num2 = scanner.nextInt();
> 
> 
>     System.out.print("Enter an operator (+, -, *, /): ");
>     char operator = scanner.next().charAt(0);
> 
>    int result = 0;
> 
>     switch(operator)
>     {
>         case '+':
>             result = num1 + num2;
>             break;
> 
>         case '-':
>             result = num1 - num2;
>             break;
> 
>         case '*':
>             result = num1 * num2;
>             break;
> 
>         case '/':
>             if (num2 !=0) {
>                 result = num1 / num2;
>             } else {
>                 result = -1;
>             }
>             break;
> 
>     }
> 
>     System.out.println(result);
> 
> }
> ```
> 
> }

Oh, I didn't realize that you sent the code as a comment. The code should be pushed in your repository and not pasted in a discription

File: README.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ Add support for calculating factorial.
 ### 6 - Mix
 
 Add support for mixing all sorts of operations.
-"5x1+2-4!" = 17
\ No newline at end of file
+"5x1+2-4!" = 17
```
Comment (Almantask) [inline README.md:50]: this is the only change. There should be more changes. Maybe you forgot to push?

**Pull request closed without merging**